### ISSUE TITLE
Change canvas background color

### ISSUE THREAD

**Issue by jakubklimek** [opendata-mvcr/ontoGrapher#233]
This is just an opinion, but I think the canvas background color should be white, not gray. Especially on low-res monitors and videocalls, the gray background looks... not so nice.

Is there any special reason for gray?

![image](https://user-images.githubusercontent.com/718827/107649198-813e7d00-6c7d-11eb-9e56-292926951723.png)
![image](https://user-images.githubusercontent.com/718827/107649236-8ac7e500-6c7d-11eb-9f61-693fddfeecc5.png)

**Comment by bindeali** [comment]
Technically, it can be any color we want (it could even be an image). It makes sense to me that the canvas "space" should be visually distinguishable from the panels, and in my admittedly completely personal opinion, I prefer the grey color (in addition, a white background probably wouldn't mesh very well with the term colors turned off, but we can obviously alter that as well). We can change the panel colors or introduce appropriate borders to the styling too.

Tagging @opendata-mvcr/c5-frontend-reviewers for others' opinions.

**Comment by karelklima** [comment]
I agree with both of you :-) We could experiment with making the grey background a little lighter, but what I think that could work best is if we switched the colors completely - do white canvas and grey panels. The diagrams would pop out visually.

As a hotfix we could do some kind of toggle to turn on white background, but having toggles is not really great, UX-wise.

**Comment by jakubklimek** [comment]
I do not think this needs to be hotfixed, just thought through. Screenshots in papers and articles need white background also and I do not see the advantage of having it gray.

**Comment by psiotwo** [comment]
The white background is reasonable IMO. Unrelated - we can consider also dark mode for our tools ...
